server/index.js

```diff
@@ -48,6 +60,81 @@ app.get('/api/yelp/:businessId', async (req, res, next) => {
   }
 });
 
+app.get('/api/reviews/:businessId', async (req, res, next) => {
+  const { businessId } = req.params;
+  const sql = `
+  select *
+    from "reviews"
+   where "storeId" = $1;
+  `;
+  const params = [businessId];
+  try {
+    const result = await db.query(sql, params);
+    res.status(200).json(result.rows);
+  } catch (err) {
+    next(err);
+  }
+});
+
+app.post('/api/reviews', async (req, res, next) => {
+  // const { userId } = req.user; --change when sign in implemented AppContext
+  const userId = 1;
+  if (!userId) {
+    throw new ClientError(401, 'invalid credentials');
+  }
+  const { businessId, imageUrl, content, drinkType, recommend } = req.body;
+  if (!drinkType) {
+    throw new ClientError(400, 'drink type is required');
+  }
+  const sql = `
+    insert into "reviews" ("userId", "storeId", "imageUrl", "content", "drinkType", "recommend")
+    values ($1, $2, $3, $4, $5, $6)
+    returning *
+    `;
+  const params = [userId, businessId, imageUrl, content, drinkType, recommend];
+  try {
+    const result = await db.query(sql, params);
+    const [reviews] = result.rows;
+    res.status(201).json(reviews);
+  } catch (err) {
+    next(err);
+  }
+});
+
+app.post('/api/uploads', uploadsMiddleware, (req, res, next) => {
+  const { caption, reviewId } = req.body;
+  if (!caption) {
+    throw new ClientError(400, 'caption is a required field');
+  }
+  const url = `/images/${req.file.filename}`;
```

Comment:
Try combining these since your form submits all of the information at the same time